### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_DeleteConfirmationPopUp_ColorContrastAnalyser: Color contrast ratio for the "Delete" button is less than required (4.5:1)

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#3988]
**EnvironmentDetails:**
Application Name:AzureNotebooks_Jyupter
Login Url: notebooks.azure.com
BrowserDetails:
Edge:Microsoft Edge 41.16299.611.0
Chrome:Version 68.0.3440.106
Operating System:Windows10
**Steps to reproduce:**
1.Login with the appropriate credentials to azure notebook account.
2.Go to Get started à libraries à Select any of the libraries à click on clone to clone the library
3.Click on Run or right click and select Run to run the notebook.
4.Navigate to run button & hit enter --> select any file --> navigate to delete button & hit enter.
5.Navigate to the Deletebutton present on the page.
6.Use ColorContrastAnalyser to check the ratio of the control.
**Actual:** 
Color contrast ratio for the "Delete" button is less than required (4.5:1)
**Expected:**
Color contrast ratio for the "Delete" button should be more than or equal to required (4.5:1)
**UserImpact:**
Low-Vision user will no information regarding the cross button present on the page.
**Attachment:**
![bugattachment3](https://user-images.githubusercontent.com/39204909/45807707-b713ed00-bce1-11e8-9b7d-92d4a8ddbf66.png)

**Comment by takluyver** [comment]
Fixed by #4679
